Sources/Features/Metrics/Model/SummedEvent.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+final class SummedEvent: Event {
+
+    internal(set) public var name: String
```

Comment:
It seems to be wrong. The class is `internal` so setter and getter are `internal` as well.